src/index.ts

```diff
@@ -28,10 +28,15 @@ const DEFAULT_CONFIG: IFormsortWebEmbedConfig = { useHistoryAPI: false };
 
 const FormsortWebEmbed = (
   rootEl: HTMLElement,
-  config: IFormsortWebEmbedConfig = DEFAULT_CONFIG
+  config: IFormsortWebEmbedConfig = DEFAULT_CONFIG,
+  style?: Partial<Pick<CSSStyleDeclaration, 'width' | 'height'>>
```

Comment:
Can you add to the `config` itself?

Otherwise the number of positional parameters here might start to grow a lot.

There's a method, `setSize`, that we should probably drop as part of this as well in favor of a `setStyle` so that it could be similarly extended in the future